$NetBSD$

CentOS 5.1 claims
bfd.texinfo:325: unknown command `colophon'
bfd.texinfo:336: unknown command `cygnus'

--- ld/ld.texinfo.orig	2013-03-25 17:06:23.000000000 +0900
+++ ld/ld.texinfo	2013-11-16 13:21:12.000000000 +0900
@@ -7863,7 +7863,7 @@ If you have more than one @code{SECT} st
 @printindex cp
 
 @tex
-% I think something like @colophon should be in texinfo.  In the
+% I think something like @@colophon should be in texinfo.  In the
 % meantime:
 \long\def\colophon{\hbox to0pt{}\vfill
 \centerline{The body of this manual is set in}
@@ -7874,7 +7874,7 @@ If you have more than one @code{SECT} st
 \centerline{{\sl\fontname\tensl\/}}
 \centerline{are used for emphasis.}\vfill}
 \page\colophon
-% Blame: doc@cygnus.com, 28mar91.
+% Blame: doc@@cygnus.com, 28mar91.
 @end tex
 
 @bye
